flink-cdc-cli/src/main/java/org/apache/flink/cdc/cli/parser/YamlPipelineDefinitionParser.java

```diff
@@ -172,6 +183,23 @@ private SinkDef toSinkDef(JsonNode sinkNode) {
         Optional.ofNullable(sinkNode.get(EXCLUDE_SCHEMA_EVOLUTION_TYPES))
                 .ifPresent(e -> e.forEach(tag -> excludedSETypes.add(tag.asText())));
 
+        if (includedSETypes.isEmpty()) {
+            // If no schema evolution types are specified, include all schema evolution types by
+            // default.
+            Arrays.stream(SchemaChangeEventTypeFamily.ALL)
+                    .map(SchemaChangeEventType::getTag)
+                    .forEach(includedSETypes::add);
+        }
+
+        if (excludedSETypes.isEmpty()
+                && SchemaChangeBehavior.LENIENT.equals(schemaChangeBehavior)) {
+            // In lenient mode, we exclude DROP_TABLE and TRUNCATE_TABLE by default. This could be
+            // overridden by manually specifying excluded types.
```

Comment:
minor: we should add schema behavior  document ASAP

Reply:
Tracked with FLINK-36151.